clickfile.py

```diff
@@ -419,6 +420,44 @@ def install_ui_requirements():
     subprocess.check_call("playwright install chromium", shell=True)
 
 
+def get_service_tags_for_cost_reports(
```

Comment:
It would be good to move all functions connected to cost report to separated module  and leave only 2 commands in clickfile as clickfile became too big